CKB-Explorer-PRDs/Glossary/Ongoing.md

```diff
@@ -0,0 +1,54 @@
+| 交易详情页 | deprecated addresses | Deprecated addresses in CKB refer to old address formats that are no longer recommended for use. These addresses were used in earlier versions of the CKB protocol, but have since been replaced by newer, more secure address formats. While deprecated addresses can still be used to send and receive transactions, it is recommended to use the newer address formats for improved security and compatibility with future versions of the CKB protocol. | checked(sven) | - | - | - |
+| Script详情页 | Script Name | Script Name is the name of a Script. A script is a data structure represents a reference to a piece of on-chain runnable code. | checked(sven) | - |
+| Script详情页 | Code Hash | Code Hash in CKB is a field in the Script data structure that represents the hash of the script code. The script code is compiled into RISC-V binary. The binary is stored as the data in a cell. When hash_type is "Data", the script locates a cell whose data's hash equals the script's code_hash.  | checked(sven) | - |
+| Script详情页 | Hash Type | Hash Type in CKB is a field in the Script that specifies how the Code Hash field should be interpreted when looking for script code to run from cell dependencies. | checked(sven) | - |
+| Script详情页 | Type Id | Type ID describes a way of using a special type script which can create a singleton type - there's only one live cell of this type. With Type ID nobody could create another code cell with the same type script hash, which makes it a useful companion to Type hash type. | checked(sven) | - |
+| Script详情页 | Capacity of deployed cells | Capacity of deployed cells in CKB refers to the maximum amount of space (in bytes) that a cell can occupy on the Nervos CKB. The capacity field is one of the four fields in the Cell data structure, and limits the size of the cell. In order to store data on the CKB, a user must hold 1 CKB for every 1 byte of space that the cell occupies. This means that the capacity of a cell directly affects the amount of CKB that a user must hold in order to store data on the network. | checked(sven) | - |
+| 地址详情页 | Live Cells | Live Cells in CKB refer to unspent cells on the blockchain that contain assets owned by users. They are similar to Unspent Transaction Outputs (UTXOs) in Bitcoin.| checked(sven) | - |
+| - | Occupied | The total occupied CKB this address have invested in Nervos DAO | checked(sven) | - |
+| - | Nervos DAO Deposit | The total CKB this address have deposited in Nervos DAO | checked(sven) | - |
+| - | Nervos DAO Compensation | The total CKB this address have received from Nervos DAO | checked(sven) | - |
+| - | Block Mined | The number of block this address has minted | checked(sven) | - |
+| - | Args| In CKB, the public key information can be stored in the args field in the script structure. | checked(sven) | - |
```

Comment:
The `args` for script execution, e.g. the `args` can be a public key hash when the script is used to verify the owner of a private key